walletrpc/service.proto

```diff
@@ -137,6 +137,26 @@ message GetAddressUtxosReplyList {
     repeated GetAddressUtxosReply addressUtxos = 1;
 }
 
+message PriceRequest {
+    // timestamp (Unix time, seconds since 1970) at which the price is being requested
+    uint64 timestamp = 1;
```

Comment:
What Timestamp should one send to get the latest available?

Reply:
If you want the latest, don't use `GetZECPrice` (which takes this timestamp argument); that's intended only for historical data. Instead, it's recommended to use `GetCurrentZECPrice`.

I suppose you could use `GetZECPrice` and specify the current time, but then you're getting the price from only one oracle; `GetCurrentZECPrice` uses up to four oracles, so it's likely to be more accurate.